### PR TITLE
Bugfix/pro 1206 volunteers page called with invalid period

### PR DESCRIPTION
**Pull request by Tschuppi81**
feriennet: Prevent crash in case of invalid period id

TYPE: Bugfix
LINK: pro-1206

The fix is in the src/onegov/feriennet/path.py file
The other changes were needed to make mypy happy in order to commit...

@Daverball Can you check the mypy changes?

### CODE REVIEW

File: src/onegov/server/cli.py [hunk 1]
```diff
@@ -231,7 +231,7 @@ def run(
             environment=sentry_environment,
             send_default_pii=send_ppi,
             traces_sample_rate=traces_sample_rate,
-            profiles_sample_rate=profiles_sample_rate,
```
Comment (Daverball) [inline src/onegov/server/cli.py:234]: This seems wrong, pretty sure it should be profiles_sample_rate, did you make sure to update your dependencies locally? Make sure to run `make install` at least once when you run into troubles with pre-commit.

Comment (Tschuppi81) [reply src/onegov/server/cli.py:234]: I followed to the init of the sentry client and there was no profiles_sample_rate. Let me do the 'make install' and check again...

**Comment by Daverball** [comment]
@Tschuppi81 Besides that the other mypy errors are probably dude to click 8.1.4, try pinning it to 8.1.3 for now. I can sort those errors out later.

File: src/onegov/server/cli.py (hunk 1 above)
Comment (Daverball) [reply src/onegov/server/cli.py:234]: https://docs.sentry.io/platforms/python/profiling/?original_referrer=https%3A%2F%2Fgithub.com%2Fgetsentry%2Fsentry-python#enable-profiling

Comment (Tschuppi81) [reply src/onegov/server/cli.py]: `make install`helped, I also can see the `profiles_sample_rate` in the sentry_sdk.init()

**Comment by Daverball** [comment]
Looks good, just need to change `click` to `click<8.1.4` in `setup.cfg` now

**Comment by Tschuppi81** [comment]
Pinning click does not work for me

```
reto@reto-pc:~/workspace/onegov-cloud$ git diff
diff --git a/setup.cfg b/setup.cfg
index 59864a181..c31a1cb01 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -69,7 +69,7 @@ install_requires =
     cached_property
     chameleon
     certifi
-    click
+    click<8.1.4
     colour
     cssmin
     cssutils
reto@reto-pc:~/workspace/onegov-cloud$ git add setup.cfg 
reto@reto-pc:~/workspace/onegov-cloud$ git commit -m "Pinning click"
trim trailing whitespace.................................................Passed
mixed line ending........................................................Passed
check for merge conflicts................................................Passed
check python ast.....................................(no files to check)Skipped
debug statements (python)............................(no files to check)Skipped
Check for nocheckin......................................................Passed
flake8...............................................(no files to check)Skipped
sass-lint............................................(no files to check)Skipped
eslint...............................................(no files to check)Skipped
bandit...............................................(no files to check)Skipped
mypy.....................................................................Failed
- hook id: system
- exit code: 1

src/onegov/server/cli.py:228: error: Unexpected keyword argument "profiles_sample_rate" for "init"; did you mean "traces_sample_rate"?  [call-arg]
src/onegov/user/models/user.py:171: error: Unused "type: ignore" comment
src/onegov/user/models/user.py:182: error: Unused "type: ignore" comment
Found 3 errors in 2 files (checked 1576 source files)

```

**Comment by Daverball** [comment]
Looks like your venv is still not up to date, you can bypass pre-commit with `--no-verify`. As long as the CI is happy it's all good. Afterwards you can try setting up your venv again.

**Pull request merged**